packages/ui/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,61 @@
+import { Slot } from '@radix-ui/react-slot'
+import { type RecipeVariants } from '@vanilla-extract/recipes'
+import clsx from 'clsx'
+import { type ComponentProps, type PropsWithChildren } from 'react'
+import { Heading, Text } from 'ui'
+import { cardHeader, cardRoot } from './Card.css'
+
+type RootStyleProps = RecipeVariants<typeof cardRoot>
+type RootProps = ComponentProps<'div'> & RootStyleProps
+
+function CardRoot({ variant, size, className, children }: PropsWithChildren<RootProps>) {
+  return <article className={clsx(cardRoot({ variant, size }), className)}>{children}</article>
+}
+
+type HeaderProps = ComponentProps<'header'>
+
+function CardHeader({ className, children }: PropsWithChildren<HeaderProps>) {
+  return <header className={clsx(cardHeader, className)}>{children}</header>
+}
+
+type MediaProps = PropsWithChildren<ComponentProps<typeof Slot>>
+
+function CardMedia({ children }: MediaProps) {
+  // Slot ensures only one element is passed as children
+  return <Slot>{children}</Slot>
+}
+
+type HeadingProps = ComponentProps<'div'>
+
+function CardHeading({ className, children }: PropsWithChildren<HeadingProps>) {
+  return <div className={className}>{children}</div>
+}
+
+type TitleProps = ComponentProps<typeof Heading>
+
+function CardTitle({ className, children }: PropsWithChildren<TitleProps>) {
+  return (
+    <Heading variant="standard.18" className={className}>
+      {children}
+    </Heading>
+  )
+}
+
+type SubtitleProps = ComponentProps<typeof Text>
+
+function CardSubtitle({ className, children }: PropsWithChildren<SubtitleProps>) {
+  return (
+    <Text color="textSecondary" className={className}>
+      {children}
+    </Text>
+  )
+}
```

Comment:
If `CardSubtitle` is just a wrapper for `Text` with a pre defined `color` props, should we spread the reaming props here? The same applies to `CartTitle`.

```suggestion
type SubtitleProps = ComponentProps<typeof Text>

function CardSubtitle({ className, children, ...delegated }: PropsWithChildren<SubtitleProps>) {
  return (
    <Text color="textSecondary" className={className} {...delegated}>
      {children}
    </Text>
  )
}

// or even

function CardSubtitle(props: PropsWithChildren<SubtitleProps>) {
  return (
    <Text color="textSecondary" {...props} />
  )
}
```

Now if for us it's very important `color` don't be modified by the consumers we can do restrict it via TS:

```suggestion
// I believe `Text` already accepts a `children` so no need to use `PropsWithChildren`
type SubtitleProps = Omit<ComponentProps<typeof Text>, 'color'>

function CardSubtitle(props: SubtitleProps ) {
  return (
    <Text color="textSecondary" {...props} />
  )
}
```

Reply:
Good point, we should spread the props, I just forgot. Will update!